### ISSUE TITLE
[Insights Hub] Redirects and connection with main Foundation site

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#5676]
**Published Hub:** 
https://foundation.mozilla.org/en/insights/ 

**InVision doc for feedback/annotations:** 
https://mozilla.invisionapp.com/share/BEZHKC3N57H#/438269978_Overview

**To do:**
- [x] First round of feedback from Insights team
- [ ] In parallel, we're working to get the progress status for the email preferences page update [as per this doc](https://docs.google.com/document/d/1cjvoU2fVqLnY7lGHNsBtrIBRTuiRZp8S3sXT6W2BPZg/edit?usp=sharing) - filed a ticket here: #5932
- [x] Insights team to update the language in the live site. Estimate time of completion: week of Dec 14th
- [x] Once step above is completed, the page is ready to go and we can do the redirects 
- [x] Move relevant URLS and set up redirects. [Redirect mapping in progress here](https://docs.google.com/spreadsheets/d/1cZL5ExYE0754budmYcTPyd2t5_RjZzHSnJuP0Shu9jM/edit?usp=sharing)
- [x] Add link to https://foundation.mozilla.org/what-we-do page

**Comment by taisdesouzalessa** [comment]
The bulk of this work is done (Insights Hub is live and connected on "What we do" Page).

We need to move solve some Wagtail issues ([rows 12-27](https://docs.google.com/spreadsheets/d/1cZL5ExYE0754budmYcTPyd2t5_RjZzHSnJuP0Shu9jM/edit?usp=sharing)) regarding nesting Data Futures and Privacy Included inside the Insights hub in the URL and in the CMS structure. Filed a ticket here #5931 since it needs discussion with Engineers as my attempts to do it by myself had failed.

Also filed another ticket so we don't forget to update the email preferences page: #5932
